CMSgov/cms-carts-seds
# MDCT-270 - Carts V3 -> Prince

# Description
* One of the 2 PRs for MDCT-270. Addresses the ability to call to prince with the print component.
* Also allows state users to post to non state path param endpoints, so they can use printPdf. Added protection for the state being required to the one other file affected.

## How to test
Checkout this branch!
https://d1corn20m4y9yl.cloudfront.net/

You should be able to print!

Use incognito or guru will break it

## Dependencies
Either use the above link for testing in AWS, or use credentials from cloudtamer -> Carts Dev
If you opt for local, you also need the host & path for prince (yes they're separate for signing purposes, I don't like arbitrary string splitting)

# Get it done
Free me from this AWS config nightmare

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
